frontend/app/src/store/blockchain/tokens.ts

```diff
@@ -39,6 +39,18 @@ export const useBlockchainTokensStore = defineStore('blockchain/tokens', () => {
     );
   };
 
+  const setState = (chain: TokenChains, data: EvmTokensRecord) => {
+    const states = {
+      [Blockchain.ETH]: ethTokens,
+      [Blockchain.OPTIMISM]: optimismTokens
+    };
+
+    set(states[chain], {
+      ...get(states[chain]),
+      ...data
+    });
+  };
```

Comment:
Rather than doing this do you think we can do the same thing we do in blockchain balances? and use a record instead of two individual refs?